### ISSUE TITLE
FR: Option to disable themed exported image dead link

### ISSUE THREAD

**Issue by ThatOtherAndrew** [zsviczian/obsidian-excalidraw-plugin#1411]
**Is your feature request related to a problem? Please describe.**
When embedding auto-exported images instead of Excalidraw drawings, I get the following comment added automatically:
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/26030336/2a1aeedb-e002-435d-a6de-fd423f2e808c)
However, I do not export themed variants of my Excalidaw drawings. These dead links add clutter to my graph views, which isn't that big a deal but would be nice to avoid.

**Describe the solution you'd like**
A boolean setting to enable or disable generating the second link in the comment would be nice.

**Additional context**
Source code permalink: https://github.com/zsviczian/obsidian-excalidraw-plugin/blob/8f96dbc21d815cc789e5359ba8a2a2f11522834b/src/main.ts#L2274-L2281

**Comment by rx-chris** [comment]
Yes!!! I am glad i'm not the only one who finds this annoying. My current workaround is to open my Vault in Visual Studio Code and used this regular expression, `, and the.+\|dark exported image\]\]` to find and remove that part of the comment in all notes. Of course that would mean I need to open my vault in VS code and scan my notes for this on a regular basis. Would be great if there is a setting to turn it off.
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/146159597/6f315f49-a538-45af-9e88-07046fe54bb1)

**Comment by zsviczian** [comment]
I will fix this in the next release. I am also including an option to omit the entire comment section. instead, I created a command palette action that if you select the embedded SVG or PNG image link in the editor then you can navigate to the corresponding excalidraw drawing without needing the links in the comment.
